tests/multipleBackend/multipartUpload.js

```diff
@@ -0,0 +1,253 @@
+const assert = require('assert');
+const AWS = require('aws-sdk');
```

Comment:
could we also do a test where versioning is enabled on scality s3 and we complete an mpu to show that it creates a new version on scality s3?